core/controllers/base_test.py

```diff
@@ -603,13 +600,9 @@ def test_html_in_translations_is_preserved_correctly(self):
                 os.path.join(os.getcwd(), 'assets', 'i18n', filename)))
             for key, value in translation_dict.iteritems():
                 tags = self._get_tags(value, key, filename)
-                if tags != master_tags_dict[key]:
-                    mismatches.append('%s (%s): %s != %s' % (
-                        filename, key, tags, master_tags_dict[key]))
-
-        # Sorting the list before printing makes it easier to systematically
-        # fix any issues that arise.
-        self.assertEqual(sorted(mismatches), [])
+                mismatches.append('%s (%s): %s != %s' % (
+                    filename, key, tags, master_tags_dict[key]))
+                self.assertEqual(tags, master_tags_dict[key], msg=mismatches)
```

Comment:
Why is this moved into the loop? It is important to be able to see _all_ the errors at once, otherwise fixing things is quite annoying (especially since we usually get the translations as bulk imports from translatewiki).

Reply:
Same reason as above

Reply:
See note above.

Reply:
I removed the test files from coverage and reverted this change